Framework/include/QualityControl/CheckRunner.h

```diff
@@ -219,7 +219,7 @@ class CheckRunner : public framework::Task
   std::string mDeviceName;
   std::map<std::string, Check> mChecks;
   std::string mDetectorName;
-  std::shared_ptr<Activity> mActivity;
+  std::shared_ptr<Activity> mActivity; // why a shared_ptr ?
```

Comment:
I think it was to share it with Checks.

Reply:
ok, thank you, I will modify the comment